### ISSUE TITLE
Exception Access Violation

### ISSUE THREAD

**Issue by MajesticGames** [FlaxEngine/FlaxEngine#2548]
**Issue description:**

I had created a 3D scene that contained several objects with custom shaders and various scripts created in visual scripting. Today, upon entering my project, modifying some script, and testing the game, Flax would close without showing any type of error. If I tested the game initially (without modifying shaders, scripts, or using any floating option windows or the menu itself), the game would run perfectly. If I made any changes (for example, changing visual options), then upon testing the game, Flax would close.

Create a new project to determine if it was my project or the engine, and in that new project everything was working correctly. I deleted the example project and returned to my project. Now, my scene does not open, and the message 'Unhandled exception: Exception_Access_Violation writing address 0x70' appears."

**Steps to reproduce:**
I don't know how to reproduce the error. I don't know what has happened since yesterday when it was working perfectly.

**Flax version:**
Flax Engine 1.81

**Comment by MajesticGames** [comment]
I have detected a problem with the next lines (in the scene file):

			"ID": "1112ee934bec58a3e1692fbbc3cf2e09",
			"TypeName": "968fcc5242e079afafd741b24d67dcd1",
			"ParentID": "799122af4d2f868db560f89ee1c5ca9a",
			"D": {
				"1aa620c24f9bce1ecf711e86e02ef44b": {
					"Type": 7,
					"Value": 200.0
				},
				"820fe6f44da8b88cf8661db6c35846f8": {
					"Type": {
						"Type": 13,
						"TypeName": "FlaxEngine.AudioClip"
					},
					"Value": "00000000000000000000000000000000"
				}

If I remove this part, flax opens the scene.

**Comment by mafiesto4** [comment]
It looks like Visual Script issue. Could you attach that asset of ID `968fcc5242e079afafd741b24d67dcd1` (in Editor use `Ctrl+O` and paste that id to open that script)?

**Comment by MajesticGames** [comment]
The shortcut ctrl+0 doesn't work, but I know which asset it is.
 
To solve the problem, I removed the ID 1112ee934bec58a3e1692fbbc3cf2e09 from the scene file and then I added (again) the script to the actor. Now the scene file works fine. I have not modified the script, it just works fine now. Here is the script:

[Movement.zip](https://github.com/FlaxEngine/FlaxEngine/files/15254247/Movement.zip)

Thank you very much.
